lib/Game.js

```diff
@@ -2,6 +2,17 @@ class Game {
   constructor(player1, player2, player3, round = 1) {
     this.players = [];
     this.currentRound = round;
+    this.puzzle = new Puzzle();
+    this.currentPuzzle = this.puzzle.puzzleArray[round-1];
+  }
+
+  nextRound() {
+    this.currentRound++;
+    this.currentPuzzle = this.puzzle.puzzleArray[this.currentRound-1];
+  }
+
+  getCategory() {
+    return this.currentPuzzle.category;
   }
```

Comment:
loved the way you found a method to get shorten our classes , and add our Round into our game class!! 👍